examples/nodejs_wav/Readme.md

```diff
@@ -0,0 +1,44 @@
+# NodeJS voice recognition example using Mozilla DeepSpeech
+
+Download the pre-trained model (1.8GB):
+
+```
+wget https://github.com/mozilla/DeepSpeech/releases/download/v0.4.1/deepspeech-0.4.1-models.tar.gz
+tar xvfz deepspeech-0.4.1-models.tar.gz
```

Comment:
We already have audio samples, why not use them instead of including another new audio file?

Reply:
There aren't many other wav files in the project and they would have to be a deep relative reference:

./examples/net_framework/CSharpExamples/DeepSpeechConsole/arctic_a0024.wav
./data/smoke_test/LDC93S1_pcms16le_1_8000.wav
./data/smoke_test/LDC93S1_pcms16le_2_44100.wav
./data/smoke_test/LDC93S1.wav

The advantage of supplying a wav file with the example, is because I bet 99% of the time someone will just want to quickly replace the audio.wav file with their own file and see what happens.  That's exactly what I did when I got mine working.

Reply:
> There aren't many other wav files in the project and they would have to be a deep relative reference:

Not sure to get what you want to mean here.